nvflare/app_common/aggregators/intime_accumulate_model_aggregator.py

```diff
@@ -12,180 +12,118 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import re
-
-from nvflare.apis.dxo import DXO, DataKind, MetaKey, from_shareable
+from nvflare.apis.dxo import DXO, DataKind, from_shareable
 from nvflare.apis.fl_constant import ReservedKey, ReturnCode
 from nvflare.apis.fl_context import FLContext
 from nvflare.apis.shareable import Shareable
 from nvflare.app_common.abstract.aggregator import Aggregator
+from nvflare.app_common.aggregators.dxo_aggregator import DXOAggregator
 from nvflare.app_common.app_constant import AppConstants
 
 
 class InTimeAccumulateWeightedAggregator(Aggregator):
     def __init__(self, exclude_vars=None, aggregation_weights=None, expected_data_kind=DataKind.WEIGHT_DIFF):
```

Comment:
Does exclude_vars apply to all keys? Or should each key have its own exclude_vars?
Similar question for aggregation_weights.

Reply:
@ZiyueXu77 @Can-Zhao what do you think?

Reply:
Each key with its own exclude_vars will give more flexibility, in case it is needed for some algorithms (although I cannot think of any at this time).

Reply:
Makes sense. I added the option.